tests/test_shipment.py

```diff
@@ -244,3 +244,42 @@ def test_generate_form(one_call_buy_shipment, rma_form_options):
 
     assert form.form_type == form_type
     assert form.form_url is not None
+
+
+@pytest.mark.vcr()
+def test_shipment_create_with_carbon_offset(carbon_offset_shipment):
+    shipment = easypost.Shipment.create(carbon_offset=True, **carbon_offset_shipment)
+
+    assert isinstance(shipment, easypost.Shipment)
+    assert shipment.rates is not None
+    assert all(rate.carbon_offset is not None for rate in shipment.rates)
+
+
+@pytest.mark.vcr()
+def test_shipment_buy_with_carbon_offset(carbon_offset_shipment):
+    shipment = easypost.Shipment.create(**carbon_offset_shipment)
+    shipment.buy(carbon_offset=True, rate=shipment.lowest_rate())
```

Comment:
Please swap these orders so that they come in order of importance. Rate first, carbon_offset second

Reply:
good catch